PluginGenerator/Maven/DataModel/MavenCoordinate.cs

```diff
@@ -0,0 +1,56 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="MavenCoordinate.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Xml.Serialization;
+
+namespace SonarQube.Plugins.Maven
+{
+    /// <summary>
+    /// Describes a Maven artifact
+    /// </summary>
+    /// <remarks>This class is XML-serializable</remarks>
```

Comment:
There are a number of simple data classes that allow us to load Maven POM files: MavenCoordinate, MavenDependency and MavenPartialPOM.